com.unity.ml-agents/Runtime/Actuators/ActuatorComponent.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEngine;
+
+namespace Unity.MLAgents.Actuators
+{
+    /// <summary>
+    /// Editor components for creating Actuators. Generally an IActuator component should
+    /// have a corresponding ActuatorComponent.
+    /// </summary>
+    public abstract class ActuatorComponent : MonoBehaviour
+    {
+        /// <summary>
+        /// Create the IActuator.  This is called by the Agent when it is initialized.
+        /// </summary>
+        /// <returns>Created IActuator object.</returns>
+        public abstract IActuator CreateActuator();
+    }
```

Comment:
I think we're going to need to expose the size (and maybe the space type) here so that we can validate that the Agent+actuators are compatible with the model.

Reply:
on `IActuator`?

Reply:
No, on ActuatorComponent. There's something similar on SensorComponent.